package.json

```diff
@@ -28,8 +28,8 @@
         "@definitelytyped/definitions-parser": "latest",
         "@definitelytyped/dtslint-runner": "latest",
         "@definitelytyped/utils": "latest",
-        "danger": "^10.1.1",
         "@octokit/rest": "^16.0.0",
+        "danger": "^10.1.1",
```

Comment:
redundant change?

Reply:
Eh, npm sorted the depedencies when I re-installed the dev deps, not rly worth reverting this change, builds take a long time.

Reply:
(Yeah, I saw that later in local tests)